config/webpack.config.base.ts

```diff
@@ -66,6 +70,13 @@ const baseConfig: webpack.Configuration = {
           },
         ],
       },
+      // Fixes a @babel/runtime resolve issue https://github.com/webpack/webpack/issues/11467#issuecomment-691873586
+      {
+        test: /\.m?js/,
```

Comment:
Moet je `mjs` ook niet als extension resolver toevoegen dan?

Reply:
Tested dev and build and it worked with only this. Doesn't appear to be necessary to be added to extension resolving.